genesis/src/lib.rs

```diff
@@ -23,8 +23,16 @@ use serde::{Deserialize, Serialize};
 pub static GENESIS_DOMAIN_ID: Lazy<DomainId> = Lazy::new(|| "genesis".parse().expect("Valid"));
 
 /// [`AccountId`] of the genesis account.
-pub static GENESIS_ACCOUNT_ID: Lazy<AccountId> =
-    Lazy::new(|| AccountId::new(GENESIS_DOMAIN_ID.clone(), "genesis".parse().expect("Valid")));
+/// TODO remove "genesis account" #4409
+pub static GENESIS_ACCOUNT_ID: Lazy<AccountId> = Lazy::new(|| {
+    AccountId::new(
+        GENESIS_DOMAIN_ID.clone(),
+        // FIXME An associated private key can exist and sign transactions even after genesis. Consider removing "genesis account" notion altogether
```

Comment:
it doesn't matter if associated private key exists because if a transaction signed by genesis is encountered after the genesis round it is rejected. So no, this cannot happen

Reply:
and I don't think that we can remove the notion of `genesis` account but maybe you have some idea

Reply:
The original motivation was that it would be better if that decision process was not necessary, but I reconsidered and closed #4409